generators/client/templates/react/src/main/webapp/app/routes.tsx.ejs

```diff
@@ -34,6 +34,17 @@ import Entities from 'app/entities';
 import PrivateRoute from 'app/shared/auth/private-route';
 import { AUTHORITIES } from 'app/config/constants';
 
+<%_ if (authenticationType !== 'oauth2') { _%>
+/*
```

Comment:
Remove the comments here, instead add some docs on the generated Readme file about this

Reply:
In Angular, we have lazy loading for admin modules, so can you do this for all the admin module components as well?

Reply:
also please test with prod profile